boards/arm/efm32pg_stk3402a/Kconfig.defconfig

```diff
@@ -16,4 +18,9 @@ config CMU_HFXO_FREQ
 config CMU_LFXO_FREQ
 	default 32768
 
+config COUNTER_GECKO_RTCC
+	bool
+	default y
+	depends on COUNTER
```

Comment:
Why is this needed? `COUNTER_GECKO_RTCC` should be enabled by the `DT_HAS_*_ENABLED` macro.

Reply:
I'm sorry, but I am not aware of the DT_HAS_*_ENABLED macro. Can you give me a link to where it is defined?

Reply:
I see it's usage below and will see if I can fix this properly